test/lib/mayaUsd/fileio/testHideOrphanedNodes.py

```diff
@@ -211,9 +223,68 @@ def testHideOnNestedVariantSwitch(self):
         self.assertTrue(pullParentVisibilityPlug[self.cPathStr].asBool())
         self.assertTrue(pullParentVisibilityPlug[self.ePathStr].asBool())
 
+    def _saveScene(self, filename):
+        cmds.file(rename=filename)
+        cmds.file(save=True, force=True, type="mayaAscii")
+    
+    def _reloadScene(self, filename):
+        cmds.file(new=True, force=True)
+        cmds.file(filename, open=True)
+
+    def testHideOnNestedVariantSwitchOnReload(self):
+        # Pull on C and E.
```

Comment:
This new test tests the JSON code at the same time since it saves and reload the orphaned nodes manager using them.